lib/ex_unit/lib/ex_unit/doc_test.ex

```diff
@@ -51,8 +50,7 @@ defmodule ExUnit.DocTest do
       iex> a + 1
       2
 
-  If you want to keep any two tests separate,
-  add an empty line between them:
+  If you want to keep any two tests separate, add an empty line between them:
```

Comment:
I think this can stay as it was previously, it will be rendered in the same way anyway.